modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -306,6 +328,32 @@ private String _formatGeneral(
 		return newContent;
 	}
 
+	private String _formatGenerics(
```

Comment:
@TamyrisTorres, due to the comment above, let's rename to `_formatTypeParameters` and also replace all content using the term `generic` by `typeParameter`. :wink: